share/spice/quandl/fundamentals/fundamentals_triggers.txt

```diff
@@ -0,0 +1,54 @@
+total assets	ASSETS
```

Comment:
Can we alpha-sort this list? (and the others?) It would make it a little easier to read/update.